src/filters/webgl_backend.class.js

```diff
@@ -19,6 +37,19 @@
     if (gl) {
       fabric.maxTextureSize = gl.getParameter(gl.MAX_TEXTURE_SIZE);
       isSupported = fabric.maxTextureSize >= tileSize;
+      var precision;
+      var precisions = ['highp', 'mediump', 'lowp'];
```

Comment:
are there outside implementation that works but do not support lowp? how do they work at all?

Reply:
I don't know actually - I just wrote it to be as safe as possible for any scenario.